### PR TITLE
Removing RunTime Postprocessor

### LINKED ISSUE #8765: Remove RunTime Postprocessor
### Description of the enhancement or error report

Deprecate and Remove the `RunTime` Postprocessor

### Rationale for the enhancement or information for reproducing the error

This Postprocessor is redundant and less capable than `PerformanceData`. Moreover, `RunTime` suffers from the bug that `PerformanceData` had fixed: #7951.

### Identified impact
(i.e. Internal object changes, limited interface changes, public API change, or a list of specific applications impacted)

A few application may have to update their input files.

### PR DESCRIPTION
**Pull request by permcody**
closes #8765

I really don't think we need to deprecate this thing. I don't think anyone really uses it in their test suite or input files. Let's see what CIVET says...

### CODE REVIEW

**Comment by dschwen** [comment]
> I really don't think we need to deprecate this thing. I don't think anyone really uses it in their test suite or input files. 

Fails 22 bison tests

**Comment by permcody** [comment]
Examples, not tests. I can switch those over to the other, better object
before this is merged.

File: test/tests/postprocessors/misc_pps/misc_pps_test.i [hunk 1]
```diff
@@ -14,16 +14,13 @@
...
-      type = ConstantIC
-      value = 0
-    [../]
   [../]
 []
 
 [AuxVariables]
   [./aux_var]
     family = SCALAR
+    initial_condition = 42
```
Comment (andrsd) [inline test/tests/postprocessors/misc_pps/misc_pps_test.i]: Why was this necessary?

Comment (permcody) [inline test/tests/postprocessors/misc_pps/misc_pps_test.i]: There's a ScalarVariable PP below that reports the value of the ScalarVariable. That's actually part of what this test is supposed to do, it shows that we can output several different Postprocessors to the Exodus file. It's actually a pretty crappy test, but that's what it does. 

The reporter was correctly reporting zero because there are no ScalarKernels in this input file but I figured that was boring. Since I had to regold it anyway, I figured I'd put a value in there so that we could "prove" that it was working properly. As you'd expect the Scalar value is correct in the initial condition and the report lags one timestep (TIMESTEP_END).

Comment (andrsd) [inline test/tests/postprocessors/misc_pps/misc_pps_test.i]: Ok, I thought you were just removing code in this patch, so it looked like a left over from something else...

**Review by andrsd — APPROVED** [review]
